### PR TITLE
Governance Poll - March 18, 2024.md

### PR DESCRIPTION
**Pull request by blimpa**
Originating forum post: https://forum.makerdao.com/t/gov12-1-2-bootstrapping-edit-proposal/23879

### CODE REVIEW

File: governance/polls/GOV12.1.2 Bootstrapping Edit Proposal - March 18, 2024.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: GOV12.1.2 Bootstrapping Edit Proposal - March 18, 2024
```
Comment (VoteWizard) [inline governance/polls/GOV12.1.2 Bootstrapping Edit Proposal - March 18, 2024.md:2]:
```suggestion
title: Stability, Support, and Governance Scope Bootstrapping Edits - March 18, 2024
```

File: governance/polls/GOV12.1.2 Bootstrapping Edit Proposal - March 18, 2024.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+summary: Signal your support or opposition to a GOV12.1.2 bootstrapping edit in order to prepare for the Launch Season of the Endgame Features.
```
Comment (VoteWizard) [inline governance/polls/GOV12.1.2 Bootstrapping Edit Proposal - March 18, 2024.md:3]:
```suggestion
summary: Signal your support or opposition to the Stability, Support, and Governance Scope bootstrapping edits in order to prepare for the Launch Season of the Endgame Features.
```

File: governance/polls/GOV12.1.2 Bootstrapping Edit Proposal - March 18, 2024.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-03-18T16:00:00
+end_date: 2024-03-21T16:00:00
+---
+# Poll: GOV12.1.2 Bootstrapping Edit Proposal - March 18, 2024
```
Comment (VoteWizard) [inline governance/polls/GOV12.1.2 Bootstrapping Edit Proposal - March 18, 2024.md:18]:
```suggestion
# Poll: Stability, Support, and Governance Scope Bootstrapping Edits - March 18, 2024
```

**Review by MakerJanSky — APPROVED** [review]
Nice work, all.

**Pull request merged**